src/App/Fossa/BuildWait.hs

```diff
@@ -0,0 +1,53 @@
+module App.Fossa.BuildWait (
```

Comment:
[nit] maybe move this to `App.Fossa.API.BuildWait`?

Reply:
Yep, that's better.